Various Updates to ValidateContent Script (#12322)
demisto/content: Packs/Base/ReleaseNotes/1_10_4.md

#### Scripts
##### ValidateContent
- Increase the script timeout to 10 minutes.

> - Add support to run validate on single content items of all types. 

Reviewer: lets mention here the docker image bump

#### Scripts
##### ValidateContent

> - Increase the script timeout to 10 minutes.

Reviewer 2: ```suggestion
- Increased the timeout for the script to 10 minutes.
```

> - Add support to run validate on single content items of all types.

Reviewer 2: ```suggestion
- Added support to run validation on single content items of all types.
```

> - Update the script's docker image tag.

Reviewer 2: ```suggestion
- Upgraded the Docker image to *demisto/xsoar-tools:1.0.0.19258*.
```
